### PR TITLE
Added gsap JavaScript library course in Hindi language

### PR DESCRIPTION
**Pull request by MOHDNEHALKHAN**
## What does this PR do?
Add a javascript animation library(GSAP) course in Hindi language

### How do we know it's really free?
Yes, it is available on YouTube

### For book lists, is it a book? For course lists, is it a course? etc.
Yes, It is a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Thenlie — CHANGES_REQUESTED** [review]
Looks like a number of valid resources were removed in this PR. Please revert those deletions. Once you have made the requested changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -467,6 +468,11 @@
 * [Web Development Course](https://www.youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n) - Apna College
 
 
+#### GSAP
+
+* [Animate anything with GSAP | Complete GSAP Course](https://www.youtube.com/playlist?list=PLbtI3_MArDOnIIJxB6xFtpnhM0wTwz0x6) - Sheryians Coding School
```
Comment (eshellman) [inline courses/free-courses-hi.md:473]:
```suggestion
* [Animate anything with GSAP \| Complete GSAP Course](https://www.youtube.com/playlist?list=PLbtI3_MArDOnIIJxB6xFtpnhM0wTwz0x6) - Sheryians Coding School
```
escape pipes

**Review by eshellman — CHANGES_REQUESTED** [review]
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. In this case, just list in the javascript main category

**Comment by MOHDNEHALKHAN** [comment]
@eshellman , please merge this PR

**Comment by eshellman** [comment]
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. In this case, just list in the javascript main category

**Comment by MOHDNEHALKHAN** [comment]
@eshellman , can i add more gsap courses in hindi language as gsap is also js library and i just want to add the courses not the books

**Comment by eshellman** [comment]
sure, 2 courses is enough for a new category

**Pull request merged**